app/assets/stylesheets/partials/_typography.scss

```diff
@@ -89,4 +89,11 @@ reduce top margin on Get Help With This Page to allow N
 */
 #content > .report-error {
     margin-top: 0;
-}
\ No newline at end of file
+}
+
+/*
+increase spacing on TNRB guidance page
+*/
+#continue-to-increasing-threshold-link {
```

Comment:
Best not to use custom css for this, especially using an id.
Should be able to move the submit button outside of the form-group and wrap it in
`<div id="action-button" class="form-field form-field--submit">`
(to separate it from the cancel link)